srcs/mainConfigFile.cpp

```diff
@@ -6,68 +6,92 @@ int main(int ac, char **av, char **env) {
-        std::cout << GRE << "Value for \"error_page\" on main: [" << errorPage << "]" << BACK << std::endl;
+                unsigned int clientMaxBodySize = it_server->first->getClientMaxBodySize();
+                std::cout << GRE << "Value for \"client_max_body_size\" on main: [" << clientMaxBodySize << "]" << BACK << std::endl;
 
-        unsigned int portRedirection = configFile.getServerData().getPortRedirection();
-        std::cout << GRE << "Value for \"port_redirection\" on main: [" << portRedirection << "]" << BACK << std::endl;
+                std::string errorPage = it_server->first->getErrorPage();
+                std::cout << GRE << "Value for \"error_page\" on main: [" << errorPage << "]" << BACK << std::endl;
 
-        //std::cout << std::endl << CYN << "Has location block: [" << configFile.getLocationData().hasLocationBlock() << "]" << BACK << std::endl << std::endl;
-        std::cout << std::endl << "Starting location block" << std::endl;
+                unsigned int portRedirection = it_server->first->getPortRedirection();
+                std::cout << GRE << "Value for \"port_redirection\" on main: [" << portRedirection << "]" << BACK << std::endl;
 
-        std::string locationRootDirectory = configFile.getLocationData().getRootDirectory();
-        std::cout << BLU << "Value for \"root_directory\" on main: [" << locationRootDirectory << "]" << BACK << std::endl;
+                /***************************************** location block data ****************************************/
+                //std::cout << std::endl << CYN << "Has location block: [" << configFile.getLocationData().hasLocationBlock() << "]" << BACK << std::endl << std::endl;
+                std::cout << std::endl << "Starting location block" << std::endl;
 
-        std::vector<AllowMethods> locationAllowMethods = configFile.getLocationData().getAllowMethods();
-        std::cout << BLU << "Value for \"allow_methods\" on main: [";
-        for (int it = GET; it != NONE; it++) {
-            switch (it) {
-                case GET:
-                    std::cout << "GET";
-                    break;
-                case POST:
-                    std::cout << "POST";
-                    break;
-                case DELETE:
-                    std::cout << "DELETE";
-                    break;
-                default:
+                /** Keep in mind that "second" is the value of the map (i.e. the data of the location block(s) inside
+                 * the current server block).
+                 * As it is a vector (so it can keep more than one server block if needed, we have to loop on this
+                 * vector to retrieve all location blocks (i.e. iterate on the it->second).
+                 * */
+                std::vector<data::Location>::const_iterator it_location = it_server->second.cbegin();//todo check cbegin
+                for (; it_location != it_server->second.end(); ++it_location) {
```

Comment:
how to access the value of the map (with the second method), which will give us the location block(s).